configure.ac

```diff
@@ -729,6 +729,11 @@ case $host in
     #AC_SUBST(OS_LINUX)
     ;;
 
+    *-cygwin*)
+    AC_MSG_RESULT([Cygwin])
+    ADDITIONAL_CFLAGS="-D_GNU_SOURCE -D_CYGWIN_ENV"
+    ;;
```

Comment:
As I understand it, `-no-undefined` must be passed in LDFLAGS and libtool will handle the DLL creation, so:

```
LDFLAGS="-no-undefined $LDFLAGS"
```

Reply:
Line added in.